pages/Discord Crash Course.md

```diff
@@ -45,81 +45,83 @@ are distinguishable from non-staff members. Professors will have a red name, Dis
 
 Discord has a lot of features that can be confusing for first-time users, here is a brief summary of some of them:
 
-<a name="pins"></a> **Pinned Messages:**
+#### <a name="pins"></a>**Pinned Messages:**
 
-In each channel, there will be a button that looks like this: <img src="https://canvas.instructure.com/users/26285220/files/123630326/preview?verifier=Rzc1UVlK6rGXZ6eetzcNOB41XrELJCbnGil63QOs" alt="A screenshot of the Discord pinned messages symbol." height="40" width="52"/>.<!--Pushpin icon in proper size--> 
+In each channel, there will be a button that looks like this: <img src="/assets/icons/pinIcon.png" alt="A screenshot of the Discord pinned messages symbol." height="40" width="52"/>.<!--Pushpin icon in proper size--> 
 This is how you access the messages that your staff has pinned. It acts almost like a message archive, so you can
 click any of the pinned messages to jump to that message or that conversation in general. Mobile users will have to
 swipe right while in the channel to see the pinned messages button.
 
-<a name="reactions"></a> **Message Reactions:**
+#### <a name="reactions"></a> **Message Reactions:**
 
 If you hover your mouse over a message (long press the message on mobile), you will see three buttons appear on the
 right side of the message. The first button allows you to react to that message with any emoji that discord has (some
 servers have custom emojis, as well). For mobile users: this button will appear at the top of the screen that pops up
 when you long press the message (it looks like a smiling emoji with a plus sign).
 
-<a name="editing"></a> **Editing a message:**
+![A gif showing a user reacting with the Cowboy emoji to a message sent by Sage][6]
+
+#### <a name="editing"></a> **Editing a message:**
 
 If you hover your mouse over a message (long press the message on mobile) that you sent, you will see three buttons
 appear on the right side of the message. The second button will look like a pencil, and you can click it to edit your
 message. When you are finished editing, you can press the enter button to save it (just press the send button on
 mobile), or the esc button to cancel the edit (press the x above the message on mobile).
 
-<a name="pinging"></a> **Pinging people:**
+![A gif showing Josh editing a typo out of a message he sent by clicking the edit icon][5]
+
+#### <a name="pinging"></a> **Pinging people:**
 
 Similarly to Twitter, Discord has a feature that allows you to @ people on the same server as you. This is called a ping,
 and it is done by putting an @ in front of someone's Discord name when sending a message. For example:
```

Comment:
Tell the user that it notifies them directly, as in it adds a (1) to their notifs bar and also pushes a notification on dekstop and mobile